download_test.html

```diff
@@ -62,13 +62,13 @@
     <script src="https://unpkg.com/@lottiefiles/lottie-player@latest/dist/lottie-player.js"></script>
     <style>
         .content {
```

Comment:
Please also change the sequence of the classes as you can see .Android_Button has lower precedence as compare to .content and .versions have higher precedence as it starts with v

Reply:
@ArunTeltia done. please check now if anything else needs to be changed.